modules/porous_flow/doc/content/source/bcs/PorousFlowSinkBC.md

```diff
@@ -0,0 +1,13 @@
+# PorousFlowSinkBC
+
+This class adds a (PorousFlowSink)[PorousFlowSink.md] and a (PorousFlowEnthalpySink)[PorousFlowEnthalpySink.md] to model adding fluid at a mass flux rate at a specified temperature.
+
+Users are required to specify:
+
+- `boundary` - The list of boundary IDs from the mesh where this boundary condition applies
+- `porepressure_var` - Name of the porepressure variable
```

Comment:
Should be

- either `fluid_phase` (the fluid phase whose porepressure is used to compute the injected enthalpy) or `porepressure_var` (an AuxVariable representing fluid pressure at which to compute the injected enthalpy)

Reply:
✅